src/utils/feature.ts

```diff
@@ -29,9 +29,10 @@ export const getFeatureValueTypeToTypeScriptMapping = (valueType: string): strin
  * Fetches all the features from the GrowthBook REST API for the provided API base URL.
  * @param apiBaseUrl  The API base URL for the target GrowthBook instance
  * @param token  Secret token
+ * @param projectId  Optional project ID filter
  * @return {Promise<SimplifiedFeature[]>} A list of features
  */
-export const fetchAllPaginatedFeatures = async (apiBaseUrl: string, token: string): Promise<SimplifiedFeature[]> => {
+export const fetchAllPaginatedFeatures = async (apiBaseUrl: string, token: string, projectId?: string): Promise<SimplifiedFeature[]> => {
```

Comment:
```suggestion
export const fetchAllPaginatedFeatures = async ({ apiBaseUrl, token,  projectId } : { apiBaseUrl: string, token: string, projectId?: string}): Promise<SimplifiedFeature[]> => {
```

Let's switch to using an object param for readibility and extensivity with future optional parameters.